packages/aragon-wrapper/src/index.js

```diff
@@ -988,6 +989,27 @@ export default class Aragon {
     })
   }
 
+  /**
+   * Initialise the path observable.
+   *
+   * @return {Promise<void>}
+   */
+  async initPath () {
+    this.path = new ReplaySubject(1)
+    this.path.next('/')
```

Comment:
This observable is the only one that I'd think a bit more on, and be careful with. A couple of thoughts:
- Having `this.path` transparently represent the current app isn't safe if a script wanted to try and use `requestPath()`.
- What we probably want is a request/response scheme, where there's an `this.pathIntents` (for sake of consistency with other "intents") that the client receives requests on, and it either accepts or rejects those intents.
- If a path change intent is accepted, we push a change to `this.path` with the proxy address and the accepted path. At any time, the client can use `setPath(proxyAddress, path)` to change the path for an app (useful, for example, when loading the org at `<org>/<voting address>/vote/3`; the client should immediately tell the voting app to load the vote once the handshake is complete).
- The `path` handler should filter out `this.path` emissions that don't match its own proxy address